nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/ResultSetRecordSet.java

```diff
@@ -128,7 +128,7 @@ protected Record createRecord(final ResultSet rs) throws SQLException {
     }
 
     @SuppressWarnings("rawtypes")
-    private Object normalizeValue(final Object value) {
+    private Object normalizeValue(final Object value) throws SQLException {
```

Comment:
Is this change necessary at all?
```suggestion
    private Object normalizeValue(final Object value) {
```

Reply:
The getArray() call is throwing this, so yes we need to rethrow the error or try catch it.
As createRecord is already handling this, and createrecord is the only place where normalizeValue is called, this looks correct to me.

Reply:
My bad, I didn't notice it was about ``java.sql.Array``